File: assignment 3/sumofdigits.cpp
#include<iostream>
using namespace std;
class Sum
{
    int num,sum=0,i;
    public : void display()
    {
        cout<<"enter a number:";
        cin>>num;
        while(num>0)
        {
            i=num%10;
            sum=sum+i;
            num=num/10;
        }
        cout<<sum;
    }
};
int main()
{
    Sum s1;
    s1.display();
}

File: JAVAAssignment 3/Sumofdigits.java
import java.util.Scanner;
class Sumofdigits
{
public static void main(String[] args)
{
int num,sum=0,i;
System.out.println("Enter a number");
Scanner sc=new Scanner(System.in);
num=sc.nextInt();
while(num>0)
{
i=num%10;
sum=sum+i;
num=num/10;
}
System.out.println(sum);
}
}